### DISCUSSION TITLE
Traefik Location?

### DISCUSSION THREAD

**Discussion by undaunt** [htpcBeginner/docker-traefik#142] [Q&A] [answered]
Hey all,

I'm moving from a single large nas/server to separate synology and server similar to Anand. Based on his Synology docker guide I was assuming Traefik is running on the Synology compose stack either in addition to, or in lieu of his NUC, but it looks like it's not in his synology compose file.

If that's the case, do we still need to rebind the nginx ports on the synology if we're just pointing to the NAS containers from a Nuc/server hosted Traefik?

Thanks.

**Comment by SimpleHomelab** [comment, marked as answer]
Hi,

I have my Traefik on NUC. On the NUC, I have rules for apps that are on my Synology. Hope this clarifies stuff.

Having said that, Traefik guide should still work on Synology with freeing up ports 80 and 443. If you are not running Traefik on Synology, you do not need to free up any ports.

**Reply by undaunt** [reply]
Cool, makes sense. Related to that, I already have working app-*.yml type rules for homebridge, pihole, etc. on my server Traefik instance, and because of that I also manually tag them in the traefik labels on cloudflare-companion. I imagine I'll just do the same as you mentioned for containers on the Synology.

However, do you need to manually tag docker containers on the synology on the NUC instance of cloudflare-companion, or do you just run multiple instances of cloudflare companion on each docker host to auto manage all the subdomain creation?

**Reply by SimpleHomelab** [reply]
Yes, you can many tag synology apps on your cf-companion on NUC. 

Alternatively, as you suggested (I have not tried). You could also run multiple cf-companion instances to auto manage the subdomains. Since cf-companion just reads the docker labels for Host rule and does not actually need traefik, you can add only the host docker label to all containers on Synology.
